test/lib/ansible_test/_internal/docker_util.py

```diff
@@ -160,6 +165,27 @@ def docker_run(args, image, options, cmd=None):
     raise ApplicationError('Failed to run docker image "%s".' % image)
 
 
+def docker_start(args, container_id, options):
+    """
+    :type args: EnvironmentConfig
+    :type container_id: str
+    :type options: list[str] | None
+    :rtype: str | None, str | None
+    """
```

Comment:
New functions should use PEP 484 type hints (comment based) and provide a description in the docstring instead.

Reply:
Done, I think :)